src/common/grpc-expr/src/error.rs

```diff
@@ -84,6 +84,12 @@ pub enum Error {
         #[snafu(backtrace)]
         source: api::error::Error,
     },
+
+    #[snafu(display("Unrecognized table option: {}", source))]
+    UnrecognizedTableOption {
```

Comment:
better carry "what" table option is not recognized

Reply:
> better carry "what" table option is not recognized

This info will be presented in `source`